src/lib.rs

```diff
@@ -1662,7 +1662,11 @@ impl Build {
                 if !(target.contains("android")
                     && android_clang_compiler_uses_target_arg_internally(&cmd.path))
                 {
-                    if target.contains("darwin") {
+                    if self.get_host()? == target {
+                        // We are not cross-compiling so we do not need to explicitly
```

Comment:
I think this is likely to break too much. We shouldn't change this for no reason.